tests/utils/dockercli/volumelifecycle.go

```diff
@@ -18,35 +18,55 @@
 package dockercli
 
 import (
-	"fmt"
+	"log"
 	"os"
 	"os/exec"
 	"strings"
+	"time"
 )
 
 // sshIdentity an array variable to prepare ssh input parameter to pass identify value
 var sshIdentity = []string{strings.Split(os.Getenv("SSH_KEY_OPT"), " ")[0], strings.Split(os.Getenv("SSH_KEY_OPT"), " ")[1], "-q", "-kTax", "-o StrictHostKeyChecking=no"}
 
 // CreateDefaultVolume is going to create vsphere docker volume with
 // defaults.
-func CreateDefaultVolume(ip string, name string) ([]byte, error) {
-	fmt.Printf("\ncreating volume [%s] on VM[%s]", name, ip)
+func CreateDefaultVolume(ip, name string) ([]byte, error) {
+	log.Printf("Creating volume [%s] on VM [%s]\n", name, ip)
 	return InvokeCommand(ip, "docker volume create --driver=vsphere --name="+name)
 }
 
+// AttachVolume - attach volume to container on given host
+func AttachVolume(ip, volName, containerName string) ([]byte, error) {
+	log.Printf("Attaching volume [%s] on VM[%s]\n", volName, ip)
+	return InvokeCommand(ip, "docker run -d -v "+volName+
+		":/vol1 --name "+containerName+
+		" busybox tail -f /dev/null")
+}
+
 // DeleteVolume helper deletes the created volume as per passed volume name.
-func DeleteVolume(name string, ip string) ([]byte, error) {
-	fmt.Printf("\ndestroying volume [%s]", name)
+func DeleteVolume(ip, name string) ([]byte, error) {
+	log.Printf("Destroying volume [%s]\n", name)
 	return InvokeCommand(ip, "docker volume rm "+name)
 }
 
-// InvokeCommand helper method can be consumed by test directly to invoke
+// KillDocker - kill docker daemon. It is restarted automatically
+func KillDocker(ip string) ([]byte, error) {
+	log.Printf("Killing docker on VM [%s]\n", ip)
+	out, err := InvokeCommand(ip, "pkill -9 dockerd")
+	time.Sleep(2 * time.Second)
+	return out, err
+}
+
+// CleanContainer - clean the container (stops and removes it)
+func CleanContainer(ip, containerName string) ([]byte, error) {
```

Comment:
`CleanContainer` sounds confusing to me .. would it be better to rename as `RemoveContainer` or "DeleteContainer" .... would prefer `RemoveContainer` instead